src/shared/models/MissionSlotGroup.ts

```diff
@@ -78,6 +106,55 @@ export class MissionSlotGroup extends Model {
     })
     public title: string;
 
+    /**
+     * Radio frequency this unit uses for comms (in MHz)
+     * @type {number}
+     * @memberOf MissionSlotGroup
+     */
+    @Attribute({
+        type: DataTypes.DOUBLE,
```

Comment:
Just noticed - since I highly doubt we'll need 15 digits of decimal precision for a radio frequency, maybe switching to `DataType.REAL` might be sufficient? `real` provides 6 digits of decimal precision according to the [PostgreSQL docs](https://www.postgresql.org/docs/9.4/static/datatype-numeric.html), which should be enough for this use case - or do we ever need more than that?

Reply:
I've even been thinking about STRING, seeing as TFAR internally uses that. In the end, there's no arithmetic being done. ^^

Reply:
Good point - can also be used for e.g. "Channel 2" or likewise!

Reply:
yes, or phone numbers (no joke).